server/permission_test.go

```diff
@@ -4,10 +4,11 @@ import (
 	"testing"
 
 	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+
 	"github.com/mattermost/mattermost-plugin-channel-export/server/pluginapi"
```

Comment:
Nit: Should the import starting with `github.com/mattermost/mattermost-plugin-channel-export` also have their own section?

Reply:
AFAIK everything beginning with `github.com/mattermost/` get grouped together by goimports.